samples/maps/series-tiledwebmap/europe-timezones-on-twm/demo.js

```diff
@@ -0,0 +1,105 @@
+(async () => {
+
+    const topology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/europe.topo.json'
+    ).then(response => response.json());
+
+    const { colors } = Highcharts.getOptions();
+
+    // Instantiate the map
+    Highcharts.mapChart('container', {
+        chart: {
+            map: topology,
+            spacingBottom: 20
+        },
+
+        title: {
+            text: 'Europe time zones on TiledWebMap'
+        },
+
+        accessibility: {
+            series: {
+                descriptionFormat: 'Timezone {series.name} with {series.points.length} countries.'
+            },
+            point: {
+                valueDescriptionFormat: '{point.name}.'
+            }
+        },
+
+        legend: {
+            enabled: false
+        },
+
+        mapNavigation: {
+            enabled: true
+        },
+
+        plotOptions: {
+            map: {
+                allAreas: false,
+                joinBy: ['iso-a2', 'code'],
+                color: 'transparent',
+                dataLabels: {
+                    enabled: true,
+                    color: '#FFFFFF',
+                    style: {
+                        fontWeight: 'bold'
+                    },
+                    // Only show dataLabels for areas with high label rank
+                    format: null,
+                    formatter: function () {
+                        if (
+                            this.point.properties &&
+                            this.point.properties.labelrank.toString() < 5
+                        ) {
+                            return this.point.properties['iso-a2'];
+                        }
+                    }
+                },
+                tooltip: {
+                    headerFormat: '',
+                    pointFormat: '{point.name}: <b>{series.name}</b>'
+                },
+                states: {
+                    hover: {
+                        borderWidth: 5
```

Comment:
With this border width, it would look good with the new `linecap` option set to `round`